### DISCUSSION TITLE
Is \Livewire\Livewire::forceAssetInjection(); supposed to override livewire.inject_assets?

### DISCUSSION THREAD

**Discussion by skeemer** [livewire/livewire#6355] [1. Help] [answered]
By its name and the way it is mentioned in the docs, I thought it would.

https://github.com/livewire/livewire/blob/0b5d87b5d0d4888a371890809a71d994ba3ce92a/src/Features/SupportAutoInjectedAssets/SupportAutoInjectedAssets.php#L16-L37

**Comment by joshhanley** [comment, marked as answer]
@skeemer yeah I would think it should. PRs welcome to fix! 🙂

**Reply by skeemer** [reply]
PR #6509 Merged

**Comment by skeemer** [comment]
@joshhanley, I noticed an incomplete test that you added called `can_disable_auto_injection_using_global_method`. Should there also be something like `\Livewire\Livewire::preventAssetInjection()`?
